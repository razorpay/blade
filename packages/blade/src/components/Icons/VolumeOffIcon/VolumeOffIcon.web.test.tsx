import VolumeOffIcon from './';
import renderWithTheme from '~utils/testing/renderWithTheme.web';

describe('<VolumeOffIcon />', () => {
  it('should render VolumeOffIcon', () => {
    const { container } = renderWithTheme(
      <VolumeOffIcon color="feedback.icon.neutral.intense" size="large" />,
    );
    expect(container).toMatchSnapshot();
  });
});
