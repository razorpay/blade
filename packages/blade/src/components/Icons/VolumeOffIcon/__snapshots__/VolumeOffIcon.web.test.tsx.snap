// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VolumeOffIcon /> should render VolumeOffIcon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="Svgweb__StyledSvg-vcmjs8-0"
    data-blade-component="icon"
    fill="none"
    height="20px"
    viewBox="0 0 24 24"
    width="20px"
  >
    <path
      clip-rule="evenodd"
      d="M2.29297 2.29289C2.6835 1.90237 3.3165 1.90237 3.70703 2.29289L19.084 17.6698C19.0877 17.6735 19.092 17.6769 19.0957 17.6806C19.0985 17.6834 19.1008 17.6866 19.1035 17.6894L21.707 20.2929L21.7754 20.3691C22.0956 20.7619 22.0731 21.3409 21.707 21.707C21.341 22.073 20.7619 22.0955 20.3691 21.7753L20.293 21.707L18.3467 19.7607C18.3404 19.7657 18.3354 19.7722 18.3291 19.7773C17.8997 20.1242 17.2701 20.0579 16.9229 19.6288C16.6155 19.2486 16.6326 18.7116 16.9385 18.3525L15.4678 16.8818C15.0402 17.1093 14.4999 16.9992 14.2002 16.5995C13.9602 16.2794 13.9414 15.8626 14.1172 15.5312L12 13.414V18.9443C12.041 19.3321 11.9565 19.7244 11.7549 20.0605C11.5338 20.4287 11.1877 20.7058 10.7803 20.8417C10.3728 20.9775 9.92999 20.9627 9.53223 20.8007C9.17097 20.6535 8.86878 20.3931 8.66895 20.0605L5.51074 15.9999H4C3.46963 15.9999 2.961 15.7889 2.58594 15.414C2.21099 15.0389 2.00007 14.5302 2 13.9999V9.99992C2.00006 9.46963 2.21101 8.96087 2.58594 8.58586C2.961 8.21085 3.46962 7.99992 4 7.99992H5.51172L5.98145 7.39543L2.29297 3.70695C1.90247 3.31645 1.90252 2.68343 2.29297 2.29289ZM6.78906 9.61418C6.5996 9.85753 6.30844 9.99992 6 9.99992H4V13.9999H6C6.30859 13.9999 6.59961 14.143 6.78906 14.3866L10 18.5146V11.414L7.40625 8.82023L6.78906 9.61418Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
      fill-rule="evenodd"
    />
    <path
      d="M16.9229 4.37102C17.2484 3.96865 17.8218 3.8849 18.2461 4.16203L18.3291 4.22258L18.6562 4.49894C20.2629 5.91545 21.3754 7.81363 21.8232 9.91594C22.301 12.1584 21.9941 14.4971 20.9531 16.54C20.7025 17.032 20.1005 17.2271 19.6084 16.9765C19.1163 16.7258 18.9202 16.1239 19.1709 15.6318C20.0037 13.9975 20.2493 12.1269 19.8672 10.3329C19.5088 8.6511 18.6193 7.13216 17.334 5.99894L17.0713 5.77727L16.9951 5.70988C16.635 5.35311 16.5974 4.77376 16.9229 4.37102Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
    />
    <path
      d="M14.2002 7.40031C14.5315 6.95862 15.1578 6.86896 15.5996 7.20012C16.4976 7.87357 17.1891 8.78535 17.5947 9.83195C18.0003 10.8786 18.1042 12.0183 17.8945 13.121C17.7913 13.6633 17.2679 14.0189 16.7256 13.9159C16.183 13.8128 15.8265 13.2896 15.9297 12.747C16.0694 12.0119 15.9999 11.2523 15.7295 10.5546C15.4591 9.85699 14.9989 9.24866 14.4004 8.79973C13.9587 8.46838 13.869 7.84211 14.2002 7.40031Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
    />
    <path
      d="M9.68359 3.14543C10.0404 3.03601 10.4237 3.04026 10.7803 3.1591C11.1369 3.278 11.4462 3.50482 11.666 3.80656L11.7549 3.94035L11.8311 4.08098C11.977 4.38205 12.0346 4.71832 12 5.05168V6.99992C11.9998 7.55207 11.5522 7.99992 11 7.99992C10.4478 7.99992 10.0002 7.55207 10 6.99992V5.48527L9.8584 5.66789L9.79102 5.74602C9.44101 6.11199 8.86352 6.16116 8.45508 5.84367C8.01935 5.50469 7.94066 4.87623 8.2793 4.44035L8.6709 3.93644C8.87075 3.60537 9.17208 3.34586 9.53223 3.19914L9.68359 3.14543Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
    />
  </svg>
</div>
`;
